src/components/personalia/fullmakt.tsx

```diff
@@ -16,7 +16,7 @@ function FullmektigEllerFullmaktsgiver(props: { fullmakt: Fullmakter }) {
             <div className="underinformasjon innrykk">
                 <Detail>
                     <b>
-                        {motpartsRolle?.substring(1).toLowerCase()}: {motpartsPersonident}
+                        F{motpartsRolle?.substring(1).toLowerCase()}: {motpartsPersonident}
```

Comment:
Skal denne F'en være her? :) 
```suggestion
                        {motpartsRolle?.substring(1).toLowerCase()}: {motpartsPersonident}
```